code/modules/food_and_drinks/food/foods/side_dishes.dm

```diff
@@ -117,6 +117,16 @@
 	tastes = list("rice" = 1)
 	goal_difficulty = FOOD_GOAL_NORMAL
 
+/obj/item/food/snacks/boiledrice/attackby(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/stack/seaweed))
+		return ..()
+	var/obj/item/stack/seaweed/S = I
+	var/obj/item/food/snacks/onigiri/O = new
+	S.use(1)
+	reagents.trans_to(O, reagents.total_volume)
+	qdel(src)
+	if(!user.put_in_active_hand(O))
+		O.forceMove(get_turf(user))
```

Comment:
```suggestion
	user.put_in_active_hand(O))
```
Unnecessary if-statement if we just make it start out on the user's turf already